platform-controller/src/main/java/org/hobbit/controller/docker/ContainerManagerImpl.java

```diff
@@ -587,16 +587,19 @@ public String startContainer(String imageName, String containerType, String pare
 
 
     @Override
-    public void removeContainer(String containerId) {
+    public void removeContainer(String taskId) {
         try {
-            Task taskInfo = dockerClient.inspectTask(containerId);
+            Task taskInfo = dockerClient.inspectTask(taskId);
+            String containerId = taskInfo.status().containerStatus().containerId();
```

Comment:
Can `containerId` be `null` there? Everything will be okay with this?

Reply:
Hmmm... It can, need to make a check for this.

Reply:
@denkv I changed the sequence, first removing container and then service, that would prevent the potential problem